Tasks/AzureAppServiceManage/azureappservicemanage.ts

```diff
@@ -104,6 +109,15 @@ async function run() {
                 if (swapWithProduction) {
                     targetSlot = "production";
                 }
+
+                sourceSlot = sourceSlot.toLowerCase();
+                targetSlot = targetSlot.toLowerCase();
+
+                if(sourceSlot == 'production') {
+                    sourceSlot = targetSlot;
+                    targetSlot = 'production';
```

Comment:
can you change 'swapWithProduction = true'.

Reply:
done